packages/suite/src/utils/suite/__tests__/protocol.test.ts

```diff
@@ -0,0 +1,18 @@
+import { getProtocolInfo, isProtocolScheme } from '@suite-utils/protocol';
+import * as fixtures from '../__fixtures__/protocol';
+
+describe('getProtocolInfo', () => {
+    fixtures.getProtocolInfo.forEach(f => {
```

Comment:
nit: would it makes sense to not use one letter variables even here? 😅 Or better stop using them altogether. OR outline where they can be use and where they shouldn't be.

Reply:
I moved it like that from another file 🤯
<img width="324" alt="Screenshot 2022-05-30 at 11 14 22" src="https://user-images.githubusercontent.com/33235762/170959435-6efd2bb7-14ff-4347-8fb7-d342e9f5b495.png">

Reply:
We have 253 occurrences. I do not think that it is a good idea to change it.

Reply:
Of course! I didn't know it's a "pattern", let it be.